wqflask/wqflask/correlation/show_corr_results.py

```diff
@@ -216,7 +216,7 @@ def __init__(self, start_vars):
             for _trait_counter, trait in enumerate(self.correlation_data.keys()[:self.return_number]):
                 trait_object = GeneralTrait(dataset=self.target_dataset, name=trait, get_qtl_info=True, get_sample_info=False)
 
-                if self.dataset.type == "ProbeSet" or self.dataset.type == "Geno":
+                if self.target_dataset.type == "ProbeSet" or self.target_dataset.type == "Geno":
```

Comment:
target_dataset is a new variable. Do we really need that? It is getting confusing...